### PR TITLE
Chore/remove-unnecesary-deps

### PR DESCRIPTION
**Pull request by kiwiidb**
Resolves https://github.com/getAlby/lndhub.go/issues/96

### CODE REVIEW

File: lnd/lnd.go [hunk 1]
```diff
@@ -15,6 +15,10 @@ import (
 	"gopkg.in/macaroon.v2"
 )
 
+const (
+	MSAT_PER_SAT = 1000
```
Comment (bumi) [inline lnd/lnd.go:19]: where is this used?

File: lib/service/invoices.go [hunk 2]
```diff
@@ -223,32 +215,19 @@ func (svc *LndhubService) PayInvoice(ctx context.Context, invoice *models.Invoic
...
-		ph := *decodedInvoice.PaymentHash
-		invoice.RHash = hex.EncodeToString(ph[:])
-	}
-	if decodedInvoice.MilliSat != nil {
-		msat := decodedInvoice.MilliSat
-		invoice.Amount = int64(msat.ToSatoshis())
+		DestinationPubkeyHex: decodedInvoice.Destination,
+		DescriptionHash:      decodedInvoice.DescriptionHash,
+		Memo:                 decodedInvoice.Description,
+		ExpiresAt:            bun.NullTime{Time: time.Unix(decodedInvoice.Timestamp, 0).Add(time.Duration(decodedInvoice.Expiry))},
```
Comment (bumi) [inline lib/service/invoices.go:230]: is a unit needed for that `time.Duration`? or is that default seconds?

**Review by bumi — APPROVED** [review]
utACK

File: lnd/lnd.go (hunk 1 above)
Comment (kiwiidb) [reply lnd/lnd.go:19]: Hmm, must have slipped in from my C-Lightning work, not used anywhere atm, deleted.

**Pull request merged**